static/src/js/actions/urlQuery.js

```diff
@@ -227,8 +227,8 @@ export const changePath = (path = '') => async (dispatch, getState) => {
     await dispatch(actions.getProjectGranules())
   }
 
-  const zustandState = useEdscStore.getState()
-  const { timeline } = zustandState
+  const zustand = useEdscStore.getState()
```

Comment:
Keep `zustandState` to be consistent across the app, or you could just grab `{ timeline }`